cli/cli.go

```diff
@@ -64,6 +61,7 @@ type CliCmdFlags struct {
 	clExtraFlags      *[]string
 	vlExtraFlags      *[]string
 	logging           string
+	trackSyncWait     int
```

Comment:
Please use `int64` instead of `int` because it's the base type of `time.Duration` and all the operations could be done without cast

Reply:
done